lib/ngx/ssl.lua

```diff
@@ -37,6 +37,9 @@ int ngx_http_lua_ffi_ssl_raw_server_addr(ngx_http_request_t *r, char **addr,
 int ngx_http_lua_ffi_ssl_server_name(ngx_http_request_t *r, char **name,
     size_t *namelen, char **err);
 
+int ngx_http_lua_ffi_ssl_client_addr(ngx_http_request_t *r, char **addr,
```

Comment:
Never mind, I'm seeing your openresty/lua-nginx-module#1145 PR now. I think you should temporarily patch `.travis.yml` file in your branch to use your lua-nginx-module branch for your openresty/lua-nginx-module#1145 PR in this PR. Otherwise it wouldn't be compatible.